src/main/web_src/src/components/fagsystem/udistub/form/partials/IkkeOppholdSammeVilkaar.tsx

```diff
@@ -0,0 +1,134 @@
+				}
+				label="Avgjørelsesdato"
+			/>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagGrunnlagOverig'
+				}
+				label="Grunnlag for avslag"
+				options={Options('avslagGrunnlagOverig')}
+				size="large"
+			/>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagGrunnlagTillatelseGrunnlagEOS'
+				}
+				label="Tillatelsesgrunnlag EOS"
+				options={Options('avslagGrunnlagTillatelseGrunnlagEOS')}
+				size="large"
+			/>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagOppholdsrettBehandlet'
+				}
+				label="Oppholdsrett behandlet"
+				options={Options('avslagOppholdsrettBehandlet')}
+			/>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagOppholdstillatelseBehandletGrunnlagEOS'
+				}
+				label="Behandlet tillatelsesgrunnlag"
+				options={Options('avslagGrunnlagTillatelseGrunnlagEOS')}
+			/>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagOppholdstillatelseBehandletGrunnlagOvrig'
+				}
+				label="Behandlet grunnlag for avslag"
+				options={Options('avslagGrunnlagOverig')}
+				size="large"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagOppholdstillatelseUtreiseFrist'
+				}
+				label="Utreisefrist"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.avslagOppholdstillatelseBehandletUtreiseFrist'
+				}
+				label="Behandlet utreisefrist"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.bortfallAvPOellerBOSDato'
+				}
+				label="Bortfall av PO eller BOS"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.tilbakeKallUtreiseFrist'
+				}
+				label="Tilbakekall utreisefrist"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.formeltVedtakUtreiseFrist'
+				}
+				label="Formelt vedtak utreisefrist"
+			/>
+			<FormikDatepicker
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.avslagEllerBortfall.tilbakeKallVirkningsDato'
+				}
+				label="Tilbakekall virkningsdato"
+			/>
+		</Kategori>
+
+		<Kategori title={'Utvist med innreiseforbud'}>
+			<FormikSelect
+				name={
+					'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.utvistMedInnreiseForbud.innreiseForbud'
+				}
+				label="Innreiseforbud"
+				options={Options('jaNeiUavklart')}
```

Comment:
Må ha en onChange på denne for å handle endring i value. For hvis man først velger JA, fyller ut varighet og vedtaksdato, for så å endre til NEI, har fortsatt varighet og vedtaksdato values i formet. Så de må nullstilles dersom value her settes til NEI eller UAVKLART.